dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/SdkCoreRegistry.kt

```diff
@@ -66,6 +69,23 @@ internal class SdkCoreRegistry(
         instances.clear()
     }
 
+    /**
+     * Wraps the core with a proxy to start listening to feature events.
+     * This must be called before any feature is registered.
+     */
+    fun wrapCoreWithProxy(name: String?): DatadogCoreProxy? {
```

Comment:
this method looks a bit weird...at least the naming. If you want to keep the same logic I would name it `try...` if not maybe the condition could be checked outside of the method in another function: `isBuildForCoreTesting` and then call this method inside:
```
coreProxy = if(isBuildForCoreTesting()){wrapWithCoreProxy(name)}
 ```